plan_delegator/src/plan_delegator.cpp

```diff
@@ -123,6 +127,7 @@ namespace plan_delegator
             auto time_diff_sec = time_diff.toSec();
             // this assumes the vehicle does not have significant lateral velocity
             plan_req.request.vehicle_state.longitudinal_vel = distance_diff / time_diff_sec;
+            // TODO develop way to set yaw value for future points
```

Comment:
Is this referring to the yaw at the end of a maneuver (as planned by an individual plugin)?